rcl/test/rcl/test_publisher.cpp

```diff
@@ -76,6 +80,34 @@ class CLASSNAME (TestPublisherFixture, RMW_IMPLEMENTATION) : public ::testing::T
   }
 };
 
+class CLASSNAME (TestPublisherFixtureInit, RMW_IMPLEMENTATION)
+  : public CLASSNAME(TestPublisherFixture, RMW_IMPLEMENTATION)
+{
+public:
+  const rosidl_message_type_support_t * ts =
+    ROSIDL_GET_MSG_TYPE_SUPPORT(test_msgs, msg, BasicTypes);
+  const char * topic_name = "chatter";
```

Comment:
All these const char * that just point to string literals and aren't changed can be `constexpr char[]` instead.

Reply:
See 5df381f

Reply:
See 5df381f